lib/wallet/rpc.js

```diff
@@ -352,6 +354,26 @@ class RPC extends RPCBase {
     return null;
   }
 
+  async rescan(args, help) {
+    if (help || args.length > 1)
+      throw new RPCError(errs.MISC_ERROR, 'rescan ( "height" )');
+
+    const valid = new Validator(args);
+    const height = valid.u32(0, 0);
+
+    await this.wdb.rescan(height);
+  }
+
+  abortrescan(args, help) {
+    if (help || args.length > 0)
+      throw new RPCError(errs.MISC_ERROR, 'abortrescan');
+
+    if (!this.wdb.rescanning)
+      throw new RPCError(errs.WALLET_ERROR, 'WalletDB is not rescanning.');
+
+    return this.wdb.abortRescan();
```

Comment:
What do you think about this instead? This is more common in the module.

Otherwise the return value is `null` because `wdb.abortRescan()` doesn't return a value

```suggestion
    this.wdb.abortRescan();
    return true;
```

Reply:
This also works! It is also more user-friendly and users will know whether the command is successfully executed or not I was doing that because as you said most of the modules return null.